sdk/core/azure-core/inc/http/curl/curl.hpp

```diff
@@ -437,6 +430,12 @@ namespace Azure { namespace Core { namespace Http {
     int64_t Length() const override { return this->m_contentLength; }
 
     int64_t Read(Azure::Core::Context const& context, uint8_t* buffer, int64_t count) override;
+
+    bool IsEOF()
```

Comment:
This is not public, right?

Reply:
it was. I just moved to be private to the Session only